tests/Support/SupportStrTest.php

```diff
@@ -191,6 +191,10 @@ public function testIs()
 
         $this->assertTrue(Str::is('foo/bar/baz', $valueObject));
         $this->assertTrue(Str::is($patternObject, $valueObject));
+
+        // Tests the wildcard parameter
+        $this->assertFalse(Str::is('foo.*.baz', 'test.bar.bar.baz', '[^.]*'));
```

Comment:
IMO, `$this->assertFalse(Str::is('foo.*.baz', 'foo.bar.bar.baz', '[^.]*'));` would be better.

`'test.bar.bar.baz'` doesn't test the `$wildcard`, it stills works without it.

Reply:
Ah, typo, I started with test.*.test or something like that and switched to foo/bar